apps/admin-gui/src/app/vos/components/application-form-list/application-form-list.component.html

```diff
@@ -231,11 +231,15 @@
           cdkDrag
           mat-row>
         </tr>
+        <tr *matNoDataRow>
+          <td [attr.colspan]="displayedColumns.length">
+            <app-alert *ngIf="this.applicationFormItems.length === 0 && !loading" alert_type="warn">
+              {{'VO_DETAIL.SETTINGS.APPLICATION_FORM.NO_APPLICATION_FORM' | translate}}
+            </app-alert>
+          </td>
+        </tr>
```

Comment:
This alert is still inside of the table. This one make sense so, if it is on purpose we can merge this.